cdap-docs/admin-manual/source/installation/azure-hdinsight.rst

```diff
@@ -0,0 +1,156 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2017 Cask Data, Inc.
+
+.. :section-numbering: true
+
+.. _admin-installation-azure-hdinsight:
+
+=========================================
+Installation on Microsoft Azure HDInsight
+=========================================
+
+Introduction
+============
+
+This section describes installing CDAP on `Microsoft Azure HDInsight
+<https://azure.microsoft.com/en-us/services/hdinsight/>`__ using
+Microsoft's website to:
+
+- Create an HBase cluster;
+- Install CDAP as an application on the cluster; and then
+- Verify that CDAP is installed and running by using the CDAP UI.
+
+Information on Microsoft Azure HDInsight is `available online
+<https://docs.microsoft.com/en-us/azure/hdinsight/>`__.
+
+CDAP |short-version| is compatible with Microsoft Azure HDInsight.
+
+
+Creating the Cluster
+====================
+
+1. Log onto Microsoft Azure and navigate to *HDInsight clusters* (under *Intelligence + analytics*):
```

Comment:
minor: Intelligence + analytics is under the "New" menu.  There should just be an "HDInsight clusters" icon on the left (at least there is on ours).